pytorch_lightning/plugins/training_type/horovod.py

```diff
@@ -116,6 +116,9 @@ def broadcast(self, obj: object, src: int = 0) -> object:
         obj = hvd.broadcast_object(obj, src)
         return obj
 
+    def post_backward(self, closure_loss: torch.Tensor, optimizer: Optimizer, opt_idx: int):
+        optimizer.synchronize()
+
```

Comment:
nice, then we should remove the hack in accelerator. There is this code:
```
if isinstance(self.training_type_plugin, HorovodPlugin):
            optimizer.synchronize()
```
we should remove it

Reply:
I removed it in this PR :)